worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -167,6 +168,10 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
       logInfo(s"Initialize HDFS support with path $hdfsDir")
       try {
         StorageManager.hadoopFs = CelebornHadoopUtils.getHadoopFS(conf)
+        if (conf.workerReuseHDFSOutputStream) {
+          StorageManager.streamsManager = new StreamsManager(conf,
+            StorageManager.hadoopFs.get(StorageInfo.Type.HDFS))
```

Comment:
Other Dfs system might gain benefits from this stream manager, maybe you can test them.